endpoints/openrtb2/auction.go

```diff
@@ -657,7 +657,7 @@ func validateNativeContextTypes(cType native1.ContextType, cSubtype native1.Cont
 		// Context is only recommended, so none is a valid type.
```

Comment:
I would appreciate feedback on the context and context sub type validation. The spec includes combinations to enforce, but also allows for exchange specific values without accompanying validation guidance. I'm interpreting this to mean exchange specific context types and context sub types are allowed in any combination.

Reply:
I like the extrapolation. I'm not sure it works that way in the wild. The linked issue is using event code 555.

Reply:
Ah ok I figured we might not have free rein to define how this works 🙂 . Event code 555 is the context type right? I don't see the associated subtypes in the linked Open Measurement spec. I guess it's best not to limit the subtypes for exchange-specific types since this is already in the wild then.

Reply:
Yeah, I like the extrapolation as well but don't think that would work for the exchange specific context types and subtypes. There doesn't seem to be any validation guidelines for those in the spec so I agree with Scott that it looks like that they are allowed in any combination